src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/jobservices/ResetNullEventTimestampMessageJobTest.java

```diff
@@ -67,87 +75,137 @@ void should_be_able_to_run_reset_null_event_timestamp_problem_message_job() {
+
+        assertEquals(messageIds, resetNullEventTimestampProblemMessageJob.run());
     }
 
     @Test
-    void should_return_json_processing_exception_when_message_content_is_incorrect() throws JsonProcessingException {
+    void should_return_json_processing_exception_when_message_content_is_incorrect()
+        throws JsonProcessingException {
+        caseEventMessageEntityMap.put("messageId", "messageId_3");
+        CaseEventMessageEntity nullEventTimestampEntity = buildMessageEntity(caseEventMessageEntityMap,
+                                                                             MessageState.UNPROCESSABLE
+        );
 
-        CaseEventMessageEntity nullEventTimestampEntity = buildMessageEntity("messageId_3", MessageState.UNPROCESSABLE);
         nullEventTimestampEntity.setCaseId("caseId_3");
         nullEventTimestampEntity.setMessageContent("{\"CaseId\":\"caseId_3\",\"EventTimeStamp\":\"ABC\"}");
 
-        when(caseEventMessageRepository.findByMessageId(messageIds))
-            .thenReturn(List.of(nullEventTimestampEntity));
-        when(objectMapper.readValue(nullEventTimestampEntity.getMessageContent(), EventInformation.class))
-            .thenThrow(jsonProcessingException);
+        when(caseEventMessageRepository.findByMessageId(messageIds)).thenReturn(List.of(nullEventTimestampEntity));
+        when(objectMapper.readValue(nullEventTimestampEntity.getMessageContent(), EventInformation.class)).thenThrow(
+            jsonProcessingException);
 
         resetNullEventTimestampProblemMessageJob.run();
 
-        assertLogMessageContains(
-            String.format("Cannot parse the message with null eventTimeStamp, message id:%s and case id:%s",
-                                               nullEventTimestampEntity.getMessageId(),
-                                               nullEventTimestampEntity.getCaseId()
-            ));
+        assertLogMessageContains(String.format(
+            "Cannot parse the message with null eventTimeStamp, message id:%s and case id:%s",
+            nullEventTimestampEntity.getMessageId(),
+            nullEventTimestampEntity.getCaseId()
+        ));
     }
 
-    private EventInformation getEventInformation() {
-        return EventInformation
-            .builder()
-            .userId("userId_3")
-            .jurisdictionId("jurisdictionId_3")
-            .caseTypeId("caseTypeId_3")
-            .caseId("caseId_3")
-            .eventTimeStamp(LocalDateTime.now())
-            .build();
+    private static EventInformation getEventInformation_one() {
+        return EventInformation.builder().userId("userId_3").jurisdictionId("jurisdictionId_3").caseTypeId(
+            "caseTypeId_3").caseId("caseId_3").eventTimeStamp(LocalDateTime.now()).build();
     }
 
-    private CaseEventMessageEntity buildMessageEntity(String id, MessageState state) {
+    private CaseEventMessageEntity buildMessageEntity(Map<String, Object> map, MessageState state) {
```

Comment:
I mentioned this to @DennisHoV1 in another review.  This should go into it's own class as you'll want to create `CaseEventMessageEntity` in multiple tests.

Reply:
Any ideas as to what that class name should be